independent-projects/tools/common/src/main/java/io/quarkus/cli/commands/file/BuildFile.java

```diff
@@ -12,11 +12,7 @@
 import io.quarkus.platform.descriptor.QuarkusPlatformDescriptor;
 import java.io.Closeable;
 import java.io.IOException;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
-import java.util.Properties;
-import java.util.TreeMap;
+import java.util.*;
```

Comment:
Star imports are not used, make sure your IDE is using the Quarkus code formatter

Reply:
Fixed